src/components/BreadcrumbsItem/BreadcrumbsItem.tsx

```diff
@@ -26,22 +39,94 @@ export interface Props {
  */
-export const BreadcrumbsItem = ({ className, text, href, ...other }: Props) => {
-  const componentClassName = clsx(styles['breadcrumbs__item'], className, {});
+export const BreadcrumbsItem = ({
+  dropdownMenuItems,
+  className,
+  href,
+  text,
+  variant,
+  ...other
+}: Props) => {
+  const [isActive, setIsActive] = React.useState(false);
+
+  /**
+   * Closes the dropdown menu if there is focus or click outside.
+   */
+  const handleBlur = (event: React.FocusEvent<HTMLElement, Element>) => {
+    if (!event.currentTarget.contains(event.relatedTarget) && isActive) {
+      setIsActive(false);
+    }
+  };
+
+  const componentClassName = clsx(
+    styles['breadcrumbs__item'],
+    variant === 'back' && styles['breadcrumbs__item-back'],
+    className,
+  );
+
+  const ellipsisButtonClassName = clsx(
+    styles['breadcrumbs__link'],
+    styles['breadcrumbs__ellipsis'],
+  );
+
+  const dropdownMenuClassName = clsx(
+    styles['breadcrumbs__dropdown-menu'],
+    isActive && styles['breadcrumbs__dropdown-menu--active'],
+  );
+
+  const getInteractionElement = () => {
+    if (variant === 'collapsed') {
+      /* The collapsed variant is a button with ellipsis. Interaction spawns a dropdown containing the collapsed breadcrumb links. */
+      return (
+        <>
+          <button
+            className={ellipsisButtonClassName}
+            onClick={() => {
+              setIsActive(!isActive);
+            }}
+          >
+            ...
+          </button>
+          <DropdownMenu
+            className={dropdownMenuClassName}
+            isActive={isActive}
+            onBlur={handleBlur}
+          >
+            {dropdownMenuItems}
+          </DropdownMenu>
+        </>
+      );
+    } else if (variant === 'back') {
+      /* The back variant is a left pointing icon that usually links to the second last breadcrumb href. */
+      return (
+        <a className={styles['breadcrumbs__link']} href={href as string}>
+          <Icon
+            className={styles['breadcrumbs__back-icon']}
+            name="chevron-left"
+            purpose="informative"
+            title={href as string}
+          />
+        </a>
+      );
+    } else {
+      /* The default variant displays the prop text and links the prop href. */
+      return (
+        <a className={styles['breadcrumbs__link']} href={href as string}>
+          {text}
+        </a>
+      );
+    }
+  };
+
   return (
     <li className={componentClassName} {...other}>
-      <a className={styles['breadcrumbs__link']} href={href}>
-        {text}
-      </a>
-      <Icon
-        className={styles['breadcrumbs__icon']}
-        color={EdsThemeColorIconNeutralStrong}
-        name="chevron-right"
-        purpose="decorative"
-        size="0.75rem"
-      />
+      {getInteractionElement()}
+      {/* The decorative separator between breadcrumbs items. */}
+      <span aria-hidden className={styles['breadcrumbs__icon']}>
```

Comment:
Maybe the class name here could be more self-explanatory so the comment isn't needed, like `breadcrumbs__separator`.

Reply:
Changed as suggested and removed comment, thanks!